calico_node/startup/startup.go

```diff
@@ -911,3 +912,26 @@ func terminate() {
 	log.Warn("Terminating")
 	exitFunction(1)
 }
+
+func fixupLoggingLevel(logLevel string) string {
+	switch strings.ToLower(logLevel) {
+	case "debug":
+		return "Debug"
+	case "info":
+		return "Info"
+	case "warning":
+		return "Warning"
+	case "error":
+		return "Error"
+	case "fatal":
+		return "Fatal"
+	default:
+		return defaultLogSeverity
```

Comment:
By default should it return the input string?

Reply:
sure, will do. each iteration is 30 minutes because of STs, so I'll probably squash as well here.